tensorflow/compiler/mlir/tensorflow/transforms/convert_tf_control_flow_to_scf.cc

```diff
@@ -41,21 +57,17 @@ class ConvertIfRegionOp : public OpRewritePattern<IfRegionOp> {
                                   Region& scf_then_or_else_region,
                                   TypeRange tf_if_region_return_type,
                                   PatternRewriter& rewriter) {
-      // Clone all the ops of `tf_then_or_else_region` into
-      // `scf_then_or_else_region`.
-      rewriter.cloneRegionBefore(tf_then_or_else_region,
-                                 &scf_then_or_else_region.front());
-      rewriter.eraseBlock(&scf_then_or_else_region.back());
-
-      Block* first_block_of_scf_then_or_else_region =
-          &scf_then_or_else_region.front();
+      // Clone the first block of `tf_then_or_else_region` into the first block
+      // of `scf_then_or_else_region` and do not add any arguments to the block.
+      cloneBlock(&tf_then_or_else_region.front(),
+                 &scf_then_or_else_region.front(), TypeRange(), rewriter);
```

Comment:
Nit: can you just replace "clone" with "move" in the doc and the function name? There is no cloning happening anymore (hopefully)

Reply:
Done.